src/mercury-common/tests/unit/task_managers/redis/test_redis_task.py

```diff
@@ -23,7 +23,7 @@ class RedisTaskTest(MercuryCommonUnitTest):
     """Tests for the mercury.common.task_managers.redis.task"""
     @mock.patch('redis.Redis')
     def setUp(self, mock_redis):
-        self.redisTask = task.RedisTask('rpc_tasks')
+        self.redisTask = task.RedisTask('rpc_tasks', None, None)
```

Comment:
Don't you mean `None, None, 'rpc_tasks' ? 

```python
 def __init__(self, redis_host, redis_port, queue_name):
```

Reply:
👍